themes/aura/assets/navbar.css

```diff
@@ -192,6 +192,9 @@
 }
 .navigation-drawer .logo img{
   max-height:40px;
+  max-width:100%;
+  width:-moz-fit-content;
```

Comment:
`width:-moz-fit-content;`
this line of code is it generate or did u put it manually?

Reply:
Auto generated